### DISCUSSION TITLE
Authentication Error, how to stay logged in

### DISCUSSION THREAD

**Discussion by wmclaxton** [community/community#23808] [New to GitHub]
<p>When I visit the page <a href="https://github.com/nextid" rel="nofollow noopener">https://github.com/nextid</a> I am prompted to ‘Sign In’.  This is despite the fact that I am signed in on another tab.  I am using login/password and Google Authenticator for 2FA.</p>
<p>Worse, if I am on a page such as <a href="https://github.com/nextid/nextcert-frontend/tree/develop" rel="nofollow noopener">https://github.com/nextid/nextcert-frontend/tree/develop</a> and there is a line which reads (for example) “Merge pull request <span class="hashtag">#21</span> from nextid/FIX/build-config”, I cannot visit the linked pull request <span class="hashtag">#21</span> without again be asked to ‘Sign In’.  When I click on that link, I am redirected to a 404 error page. Once I click the ‘Sign In’ button, I am immediately able to see the pull request.</p>
<p>There is clearly an authentication problem and it’s killing my productivity!  How do I stay logged in?</p>
<p>PS - This may or may not be related to BitDefender Anti-Tracker.</p>

**Comment by rshzl** [comment, marked as answer]
<p>Hi there,</p>
<p>I’ve been encountering the same issue on my end. I’m using Google Chrome (83.0.4103.61) on macOS Catalina (10.15.4).</p>
<p>As you mentioned at the very bottom of your post, the issue seems to be related to the <strong>BitDefender Anti-Tracker</strong> extension.</p>
<p>I ended up trying to debug this issue earlier this morning.</p>
<p>Disabling protection for <a href="http://github.com" rel="nofollow noopener">github.com</a> doesn’t seem to fix the issue for me. The only thing that’s worked so far is just outright disabling the extension.</p>
<p>Hope this helps!</p>

**Comment by wmclaxton** [comment]
<p>Exactly, I discovered the same thing.  In addition, it was causing me problems with Google Calendar and Drive.  I was creating exceptions that were partially but not fully successful. So I have disabled BitDefender Anti-Tracker.</p>
<p>The issue may be considered SOLVED.</p>
